features/visual_studio/chat/build.bat

```diff
@@ -0,0 +1,9 @@
+@echo off
```

Comment:
Put contents inside a folder that is called after the "feature". Something like:

features/visual_studio/creating_consuming_packages/chat...
features/visual_studio/msbuild_tasks/....


It is the only way to scale examples without creating confusiong

Reply:
Done